lib/liquid_voting/release.ex

```diff
@@ -11,6 +14,32 @@ defmodule LiquidVoting.Release do
     {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, :down, to: version))
   end
 
+  @doc """
+  Deletes votes, participants and delegations for the smoke tests organization_id.
+  For use before and after running smoke tests.
+  """
+  def smoke_test_teardown() do
+    for repo <- repos() do
+      {:ok, _, _} = Ecto.Migrator.with_repo(repo, fn _repo -> run_teardown() end)
+    end
+  end
+
+  defp run_teardown() do
+    @test_organization_id
```

Comment:
It'd be good to have some output to sure we know this ran sucessfully. Maybe a basic count of resources before the teardown runs, and another after? Output could be:

```
## About to run smoke test teardown 
Current resource counts: 
10 Participants, 5 Votes, 3 Delegations
## Running smoke test teardown on test organization data
## Teardown ran successfully.
End resource counts: 
0 Participants, 0 Votes, 0 Delegations
## Have a nice day
```

Reply:
sure.

Reply:
After running all 4 smoke tests, `eval "LiquidVoting.Release.smoke_test_teardown"` now outputs: 
![teardown-message](https://user-images.githubusercontent.com/3944042/88555131-4a758200-d01f-11ea-86e8-0cee1088a381.png)

Reply:
🍭 sweet

Reply:
Love the polite code ❤️